bioptim/dynamics/problem.py

```diff
@@ -578,6 +579,11 @@ def configure_dynamics_function(ocp, nlp, dyn_func):
 
         nlp.parameters = ocp.v.parameters_in_list
         nlp.p = ocp.v.parameters.cx
+
+        if nlp.parameters.__len__():
+            nlp.p_scaling = np.vstack([p.scaling for p in nlp.parameters])
+        else:
+            nlp.p_scaling = ocp.cx(1)
```

Comment:
Why if no len then 1? why not if no len then nothing?

Reply:
because this way, the scaling operation is systematically done (even when not changing the variables), and we save multiple tests everywhere